### PR TITLE
Fix injecting settings on Windows machine

### PR DESCRIPTION
**Pull request by annakocheshkova**
Will fix an error introduced in #393

### CODE REVIEW

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid setting
 
         if (!File.Exists(loaderZipFile))
         {
-            Debug.LogWarning("Failed to load dependency file appcenter-loader-release.aar");
+            throw new Exception("Failed to load dependency file appcenter-loader-release.aar");
```
Comment (Jamminroot) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: IO?

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 2]
```diff
@@ -79,7 +79,7 @@ public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid setting
         AndroidLibraryHelper.UnzipFile(loaderZipFile, loaderFolder);
         if (!Directory.Exists(loaderFolder))
         {
-            Debug.LogWarning("Unzipping loader folder failed.");
+            throw new Exception("Unzipping loader folder failed.");
```
Comment (Jamminroot) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: Same

**Review by Jamminroot — CHANGES_REQUESTED** [review]

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 3]
```diff
@@ -66,7 +66,7 @@ public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid setting
 
         if (!File.Exists(loaderZipFile))
         {
-            Debug.LogWarning("Failed to load dependency file appcenter-loader-release.aar");
+            throw new Exception("Failed to load dependency file appcenter-loader-release.aar");
             return;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: `return` can be removed now

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 4]
```diff
@@ -79,7 +79,7 @@ public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid setting
         AndroidLibraryHelper.UnzipFile(loaderZipFile, loaderFolder);
         if (!Directory.Exists(loaderFolder))
         {
-            Debug.LogWarning("Unzipping loader folder failed.");
+            throw new Exception("Unzipping loader folder failed.");
             return;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: Same

File: Assets/AppCenter/Plugins/Android/Utility/archive.ps1 [hunk 5]
```diff
@@ -0,0 +1,18 @@
+param
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/Android/Utility/archive.ps1:4]: Copyright

File: Assets/AppCenter/Plugins/Android/Utility/unarchive.ps1 [hunk 6]
```diff
@@ -0,0 +1,20 @@
+param
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/Android/Utility/unarchive.ps1:4]: Same

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by Jamminroot — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by olmisa — APPROVED** [review]

**Pull request merged**